src/components/Search/Search.js

```diff
@@ -1,29 +1,100 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import SearchIcon from '@material-ui/icons/Search';
 import InputBase from '@material-ui/core/InputBase';
-
 import { FormattedMessage } from 'react-intl';
+import { Link, useHistory } from 'react-router-dom';
 import useStyles from './searchStyles';
+import fuse from './FuseSearch';
 
 function Search() {
   const classes = useStyles();
+  const history = useHistory();
+  const [searchQuery, setSearchQuery] = useState('');
+  const [showResults, setShowResults] = useState(false);
+  const [searchResults, setSearchResults] = useState([]);
+  const handleChange = event => {
+    setSearchQuery(event.target.value);
+  };
+
+  useEffect(() => {
+    const results = fuse.search(searchQuery).slice(0, 15);
+    setSearchResults(results);
+  }, [searchQuery]);
+
+  const handleSearch = e => {
+    e.preventDefault();
+    const potentialCategory = searchResults.length ? searchResults[0].item.category : '';
+    setShowResults(false);
+    history.push(`/search?q=${searchQuery}&c=${potentialCategory}&pr=0-0`);
```

Comment:
Can we use complete words? like **query**, **category** and also I don't know what do you mean by pr=0-0